### PR TITLE
EN-3677 Modify Filter AWS Untagged EC2 Instances Lego

### PR DESCRIPTION
**Pull request by shloka-bhalgat-unskript**
## Description
1. Added try-except block to iterate over multiple regions
2. Added action_is_check as true

### Testing
Screenshot 2022-12-12 at 6 23 20 PM

### Checklist:
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

File: AWS/legos/aws_filter_ec2_without_lifetime_tag/aws_filter_ec2_without_lifetime_tag.py [hunk 1]
```diff
@@ -39,20 +39,22 @@ def aws_filter_ec2_without_lifetime_tag(handle, lifetime_tag: str, region: str)
         :rtype: Array of instances which not having lifetime tag.
     """
 
+    untagged_instances_dict={}
     ec2Client = handle.client('ec2', region_name=region)
```
Comment (jayasimha-raghavan-unskript) [inline AWS/legos/aws_filter_ec2_without_lifetime_tag/aws_filter_ec2_without_lifetime_tag.py:43]: Are you checking if region is `''` here? Does handle.client take region_name as empty quotes?

**Review by jayasimha-raghavan-unskript — CHANGES_REQUESTED** [review]
Please see my comment, It may be a good idea to check if the region is empty (`''`) and then take action appropriately.

File: AWS/legos/aws_filter_ec2_without_lifetime_tag/aws_filter_ec2_without_lifetime_tag.py (hunk 1 above)
Comment (shloka-bhalgat-unskript) [reply AWS/legos/aws_filter_ec2_without_lifetime_tag/aws_filter_ec2_without_lifetime_tag.py:43]: Oh. It doesn't. Maybe I can just give an empty List in the runbook itself and remove  `' '`

Comment (shloka-bhalgat-unskript) [reply AWS/legos/aws_filter_ec2_without_lifetime_tag/aws_filter_ec2_without_lifetime_tag.py:43]: Made the changes. Please re-review

**Pull request closed without merging**